frontend/hundru-fall/hundru.css

```
@@ -0,0 +1,2822 @@
+    0 8px 30px rgba(0, 0, 0, 0.15);
+
+  backdrop-filter: blur(12px);
+}
+
+.navbar {
+  width: min(
+    calc(100% - 40px),
+    1250px
+  );
+
+  height: 78px;
```

Stylelint (17.14.0)

[error] Expected no empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)